src/providers/TxHistoryProvider.tsx

```diff
@@ -81,41 +119,15 @@ export default function TxHistoryProvider({
       console.info('TxHistoryProvider::poller::polling for tx updates...')
 
       const transactionLogs = await Promise.all(
-        transactions.map(async txLog => {
-          // Only do refresh logic for pending txs
-          // tx.hash shouldn't ever be undefined but it's optional typed :shrug:
-          if (!txLog.tx.hash || txLog.status !== TxStatus.pending) {
-            return txLog
-          }
-
-          // If no response yet, get response.
-          // We know tx is a TransactionResponse if `.wait` is defined.
-          const response = (txLog.tx as TransactionResponse).wait
-            ? (txLog.tx as TransactionResponse)
-            : await readProvider.getTransaction(txLog.tx.hash)
-
-          let status = TxStatus.pending
-          try {
-            // Tx has been mined
-            status = TxStatus.success
-          } catch (_) {
-            // ethers provider throws error when a transaction fails
-            status = TxStatus.failed
-          }
```

Comment:
I don't understand what this was doing prviously. Won't the `catch` never be possible? 🤔

Reply:
It's possible code has been removed